src/styles/card.css

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 200px;
+  padding: 16px;
+  padding-top: 32px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 100%;
+  padding: 0 3px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-top: 37px;
```

Comment:
just curios why its 37px?

Reply:
I was trying to find the perfect match so that all tests could be run successfully :)